Makefile

```diff
@@ -608,9 +608,8 @@ test-postgres: test-clean test-postgres-docker
 		--jsonfile="gotests.json" \
 		--packages="./..." -- \
 		-covermode=atomic -coverprofile="gotests.coverage" -timeout=20m \
-		-parallel=4 \
 		-coverpkg=./... \
-		-count=1 -race -failfast
```

Comment:
Why remove? I think it forces go test to not skip cached tests.

Reply:
Go test caching is pretty effective in reducing only duplicates, we use it for the other jobs